manager/src/main/java/org/openremote/manager/system/HealthService.java

```diff
@@ -63,7 +63,7 @@ public void onInit(Route route) {
 
     public static final System.Logger LOG = System.getLogger(HealthService.class.getName());
     public static final String OR_METRICS_PORT = "OR_METRICS_PORT";
-    public static final int OR_METRICS_PORT_DEFAULT = 8404;
+    public static final int OR_METRICS_PORT_DEFAULT = 8403;
```

Comment:
Don't want the default changed here, you can override within your own Run configuration as needed

Reply:
Accidentally committed the file, but for some reason something is already hosted on 8404 at least on my PC, and I'm not exactly sure what